libs/agentc_cli/agentc_cli/cmds/ls.py

```diff
@@ -0,0 +1,35 @@
+import click
+import couchbase.cluster
+import typing
+
+from agentc_cli.cmds.util import DASHES
+from agentc_cli.cmds.util import KIND_COLORS
+from agentc_cli.cmds.util import get_catalog
+from agentc_cli.models import Context
+
+
+def cmd_ls(
+    ctx: Context,
+    kind: list[typing.Literal["tool", "prompt"]],
+    bucket: str = None,
+    cluster: couchbase.cluster.Cluster = None,
+    force_db=False,
+):
+    if ctx is None:
+        ctx = Context()
+
+    for k in kind:
```

Comment:
```suggestion
    for kind in kind_list:
```
rename `kind` variable to `kind_list`